static/docs/command-reference/get-url.md

```diff
@@ -24,8 +24,9 @@ location into the current working directory, regardless of whether it's a DVC
 project. The `dvc get-url` command helps the user do just that.
 
 The `url` argument should provide the location of the data to be downloaded,
-while `out` can be used to specify the (path and) file name desired for the
-downloaded data file or directory.
+while `out` can be used to specify the directory and/or file name desired for
+the downloaded data file or directory. If an existing directory is specified
+(without a file name), then the output will be placed inside of it.
```

Comment:
We can shorten to "...directory and/or file name desired for the downloaded data. If an existing directory is specified, then..."